### DISCUSSION TITLE
Fastdds Participant shall publish process name

### DISCUSSION THREAD

**Discussion by trittsv** [eProsima/Fast-DDS#5147] [Support]
### Is there an already existing issue for this?

- [X] I have searched the existing issues

### Expected behavior

Property(key="__ProcessName", value="DDSHelloWorldExample")

### Current behavior

DcpsParticipant(
    key=UUID('010fe616-201f-c1bd-0000-0000000001c1'),
    qos=Qos(
        Policy.EntityName(name='Participant_pub'),
        Policy.Liveliness.Automatic(lease_duration=20000000000),
        Property(key="PARTICIPANT_TYPE", value="SIMPLE"),
        Property(key="__NetworkAddresses", value="udp/192.168.0.44:7410@7"),
        Property(key="fastdds.physical_data.host", value="trittsv-desktop:6303166390786064384"),
        Property(key="fastdds.physical_data.process", value="7968"),
        Property(key="fastdds.physical_data.user", value="sven")
    )
)

### Steps to reproduce

Run Hello World App.

### Fast DDS version/commit

2.14.3

### Platform/Architecture

Windows 10 Visual Studio 2019

### Transport layer

UDPv4

**Comment by JesusPoderoso** [comment]
Hi @trittsv, thanks for using Fast DDS.
As you point out, you can configure [physical data through properties](https://fast-dds.docs.eprosima.com/en/latest/fastdds/property_policies/non_consolidated_qos.html#physical-data-in-discovery-information) for statistics purposes (regarding the use of the [Fast DDS Monitor](https://fast-dds-monitor.readthedocs.io/en/latest/index.html)).
Unfortunately, the process is identified with the process ID, not with any name.

This property `Property(key="__ProcessName", value="DDSHelloWorldExample")` will not be handled by Fast DDS.
We would need more information about the issue you described.

Also, I am moving this issue to the Support forum according to the [Fast DDS CONTRIBUTING guidelines](https://github.com/eProsima/Fast-DDS/blob/master/CONTRIBUTING.md).
